Wikipedia/Code/WMFAlertManager.swift

```diff
@@ -34,18 +34,6 @@ open class WMFAlertManager: NSObject, RMessageProtocol, Themeable {
         super.init()
         RMessage.shared().delegate = self
     }
-    
-    
-    @objc func showInTheNewsAlert(_ message: String?, sticky:Bool, dismissPreviousAlerts:Bool, tapCallBack: (() -> Void)?) {
-        
-        if (message ?? "").isEmpty {
-            return
-        }
-        showAlert(dismissPreviousAlerts, alertBlock: { () -> Void in
-            let title = CommonStrings.inTheNewsTitle
-            RMessage.showNotification(in: nil, title: title, subtitle: message, iconImage: UIImage(named:"trending-notification-icon"), type: .normal, customTypeName: nil, duration: sticky ? -1 : 2, callback: tapCallBack, buttonTitle: nil, buttonCallback: nil, at: .top, canBeDismissedByUser: true)
```

Comment:
If I'm not mistaken, `trending-notification-icon` doesn't appear to be used anywhere else. Perhaps we should remove it from the asset catalog as well.

Reply:
@staykids Great catch! This has been removed.